var/spack/repos/builtin/packages/pika/package.py

```diff
@@ -104,6 +104,12 @@ class Pika(CMakePackage, CudaPackage, ROCmPackage):
     patch("transform_mpi_includes.patch", when="@0.3.0 +mpi")
     patch("mimalloc_no_version_requirement.patch", when="@:0.5 malloc=mimalloc")
 
+    # Fix missing template instantiation on macOS
+    patch(
+        "https://github.com/pika-org/pika/commit/dd1dfb85781ec2e76fa37ce7311323e69fbe42a1.patch?full_index=1",
+        when="@0.7.0 platform=darwin",
```

Comment:
I was testing it and I learnt that patches from URLs require also checksums.

You should add `sha256=2944f746f5ae4385aba11b7c4a2f991abc108b08ea3dc394bf61c20fc7a2c4f2` as parameter of the patch function.

Reply:
Ah, I was also not aware. Thanks for checking! Does it look better now?

Reply:
It looks good!